src/metabase/query_processor/pivot.clj

```diff
@@ -417,7 +419,7 @@
                                (lib.metadata.jvm/application-database-metadata-provider (:database query)))
         query              (lib/query metadata-provider query)
         index-in-breakouts (into {}
-                                 (comp (filter (comp #{:source/breakouts :source/aggregations} :lib/source))
+                                 (comp (filter (some-fn :lib/breakout? #(= (:lib/source %) :source/aggregations)))
```

Comment:
I think letting any of these `:lib/source` enums leak out of the lib is dropping the ball on the lib's value prop. The QP should not care how the lib stores these details, it should care about `(lib/breakout? some-col)` and so on.

Let's make new predicates for these and ban direct references to `:lib/source` and its values outside of the lib? Can be a separate PR.